htdocs/language/english/locale.php

```
@@ -69,17 +77,21 @@ public function number_format($number)
     public function money_format($format, $number)
     {
-        if (!extension_loaded('intl')) {
-            return '$' . number_format((float)$number, 2, '.', ',');
-        }
+        $c = self::CURRENCY;
 
+        if (extension_loaded('intl')) {
         static $fmt = null;
         if (null === $fmt) {
-            $fmt = new \NumberFormatter('en_US', \NumberFormatter::CURRENCY);
+                $fmt = new \NumberFormatter($c['locale'], \NumberFormatter::CURRENCY);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '85', column '28'). (undefined)

(MissingImport)